src/cf.data.pre

```diff
@@ -2915,21 +2930,21 @@ DOC_START
 				      Be warned that this reduces SSL/TLS
 				      strength to some attacks.
 
-				See the OpenSSL SSL_CTX_set_options documentation
-				for a more complete list.
+			see https://wiki.openssl.org/index.php/List_of_SSL_OP_Flags
+			for a more complete list. Options relevant only to SSLv2 are
+			not supported.
 
-			GnuTLS options most important are:
+			For GnuTLS these are a list of priority strings.
+			The most important being:
 
-			    %NO_TICKETS
+				%NO_TICKETS
 				      Disable use of RFC5077 session tickets.
 				      Some servers may have problems
 				      understanding the TLS extension due
 				      to ambiguous specification in RFC4507.
 
-				See the GnuTLS Priority Strings documentation
-				for a more complete list.
-				http://www.gnutls.org/manual/gnutls.html#Priority-Strings
-
+			see https://www.gnutls.org/manual/gnutls.html#Priority-Strings
+			for a more complete list and documentation.
```

Comment:
Please either drop `and documentation` from or add it to all similar phrases elsewhere in this PR, for consistency sake. I suggest the former because it is simpler:
```suggestion
			for a more complete list.
```

Reply:
Done